packages/web-service/package.json

```diff
@@ -13,7 +14,7 @@
   },
   "scripts": {
     "postinstall": "node_modules/gulp/bin/gulp.js build",
-    "start": "node src/web-service.js",
+    "start": "nodemon -e js,njk,json -r esm src/web-service.js",
```

Comment:
I think this is fine as the container will start using `node src/web-service.js`

Reply:
Yes I was thinking about this one, it would be nice to have a `start:dev`, `start:container` - wording negotiable. Then if we need to build we have the relevant `build:prod` or `build:container` that will minify all the assets for testing and/or deployments.

Reply:
I had some esm transpiling trouble; sometime it worked sometimes it didn't so will try and fix that for the next PR. But it works in Sonar which is the main thing.